src/main/resources/assets/natura/lang/de_DE.lang

```diff
@@ -0,0 +1,582 @@
+##########################
+#    Overworld Blocks    #
+##########################
+
+tile.natura.clouds.white.name=Wolke
+tile.natura.clouds.white.tooltip=So leicht und flauschig!\nRettet Dich, wenn Du fällst
+tile.natura.clouds.dark.name=Dunkle Wolke
+tile.natura.clouds.dark.tooltip=So leicht und flauschig!\nVerursacht Blitzeinschläge
+tile.natura.clouds.ash.name=Aschewolke
+tile.natura.clouds.ash.tooltip=So leicht und flauschig!\n3x3 Verwandelt sich in Holzkohle
+tile.natura.clouds.sulfur.name=Schwefelwolke
+tile.natura.clouds.sulfur.tooltip=So leicht und flauschig!\n2x2 Verwandelt sich in Schwefel
+
+tile.natura.overworld_logs.maple.name=Ahornholz
+tile.natura.overworld_logs.maple.tooltip=Etwas süßlich
+tile.natura.overworld_logs.silverbell.name=Silberglöckchenholz
+tile.natura.overworld_logs.silverbell.tooltip=Silberne Glocken
+tile.natura.overworld_logs.amaranth.name=Amaranthholz
+tile.natura.overworld_logs.amaranth.tooltip=Herz aus Holz
+tile.natura.overworld_logs.tiger.name=Tigerholz
+tile.natura.overworld_logs.tiger.tooltip=Wildgetreide
+
+tile.natura.overworld_logs2.willow.name=Weidenholz
+tile.natura.overworld_logs2.willow.tooltip=Der Weinende
+tile.natura.overworld_logs2.eucalyptus.name=Eukalyptusholz
+tile.natura.overworld_logs2.eucalyptus.tooltip=Das rosa Holz
+tile.natura.overworld_logs2.sakura.name=Sakuraholz
```

Comment:
The name of this plant is "Japanische Blütenkirsche", but I think "Blütenkirsche" would suffice here.

Reply:
Uh, you're right, I'll correct it.